bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/engine/CacheContent.java

```diff
@@ -0,0 +1,115 @@
+package com.adobe.acs.commons.httpcache.engine;
+
+import com.adobe.acs.commons.httpcache.exception.HttpCacheDataStreamException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.*;
+
+/**
+ * Represents response content to be cached.
+ */
+public class CacheContent {
+    private static final Logger log = LoggerFactory.getLogger(CacheContent.class);
+
+    /** Response character encoding */
+    private String charEncoding;
+    /** Response content type */
+    private String contentType;
+    /** Response headers */
+    private Map<String, List<String>> headers = new HashMap<>();
+    /** Response content in file */
+    private InputStream dataInputStream;
```

Comment:
WDYT about capturing the size of the content? Generally thinking it would be interesting to know how big the total cache is.